easybuild/easyconfigs/l/libharu/libharu-2.3.0-foss-2021a.eb

```
@@ -0,0 +1,42 @@
+# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
+#
+# Copyright:: Copyright 2012-2014 Cyprus Institute / CaSToRC, Uni.Lu/LCSB, NTUA
+# Authors::   George Tsouloupas <g.tsouloupas@cyi.ac.cy>, Fotis Georgatos <fotis@cern.ch>
+# License::   MIT/GPL
+# $Id$
+#
+# This work implements a part of the HPCBIOS project and is a component of the policy:
+# http://hpcbios.readthedocs.org/en/latest/HPCBIOS_2012-94.html
+##
+# Modified for foss-2016b by:
+# Adam Huffman
+# The Francis Crick Institute
+# foss-2020b modified by:
+# Sebastien Moretti
+# SIB Swiss Institute of Bioinformatics
+
+easyblock = 'CMakeMake'
+
+name = 'libharu'
+version = '2.3.0'
+
+homepage = 'http://libharu.org/'
+description = """libHaru is a free, cross platform, open source library for generating PDF files."""
+
+toolchain = {'name': 'foss', 'version': '2021a'}
+
+source_urls = ['https://github.com/libharu/libharu/archive/']
+sources = ['RELEASE_%s.tar.gz' % '_'.join(version.split('.'))]
+
+dependencies = [('libpng', '1.6.37', '', ('GCCcore', '10.3.0'))]
+
+builddependencies = [('CMake', '3.20.1', '', ('GCCcore', '10.3.0'))]
```

@smoretti You shouldn't hardcode the subtoolchain to be used for dependencies in easyconfigs you contribute back. Also, we slightly prefer having build dependencies listed before dependencies (that's just a style issue, it has no impact on the installation):

```suggestion
builddependencies = [('CMake', '3.20.1')]

dependencies = [('libpng', '1.6.37')]

---

Ah, I messed up the suggestion a bit, since I didn't remove the 2nd `dependencies = ` line, sorry...

---

Done